src/main/java/at/fh/joanneum/irfc/service/VotingPartialResultService.java

```diff
@@ -0,0 +1,30 @@
+package at.fh.joanneum.irfc.service;
+
+import at.fh.joanneum.irfc.model.event.VotingPartialResultDTO;
+import at.fh.joanneum.irfc.model.event.VotingPartialResultMapper;
+import at.fh.joanneum.irfc.persistence.entiy.VotingPartialResultEntity;
+import at.fh.joanneum.irfc.persistence.repository.VotingPartialResultRepository;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+/**
+ * @author https://github.com/GoldNova
+ **/
+@RequestScoped
+public class VotingPartialResultService {
+    @Inject
+    VotingPartialResultRepository votingPartialResultRepository;
+    @Transactional
+    public VotingPartialResultDTO create(VotingPartialResultDTO votingPartialResultDTOCreate) {
+        VotingPartialResultEntity newEntity = new VotingPartialResultEntity();
+        setValues(votingPartialResultDTOCreate, newEntity);
+        votingPartialResultRepository.persist(newEntity);
+        return VotingPartialResultMapper.INSTANCE.toDto(newEntity);
+    }
+    private static void setValues(VotingPartialResultDTO votingPartialResultDTOCreate, VotingPartialResultEntity newEntity) {
```

Comment:
Instead of having this setter, there are two better options:

a.) I would recommend creating a Constructor in `VotingPartialResultEntity` with those needed parameters
b.) Alternatively, this could be changed to a converter where the initialization is done within this method which then returns the object

Reply:
I have followed the example of @PhilipMoser . I guess it is good to have the setter logic like this for now in order to have the partial db implemented (not all needed entities are definded in DB yet)

Reply:
@marioradman 
sugesstion a is not recommendable because it leads to a unnecessary nonNull-constructor. that will sugesst to set values in other context. 
sugesstion b is worse, because you will lose the context of the database object. and so you will not be able to  process transactional actions